WordPress/src/main/java/org/wordpress/android/ui/posts/EditPostSettingsFragment.java

```diff
@@ -1024,6 +1011,27 @@ private void showTagsActivity() {
         startActivityForResult(tagsIntent, ACTIVITY_REQUEST_CODE_SELECT_TAGS);
     }
 
+    private void showStatusDialog() {
+        AlertDialog.Builder builder = new AlertDialog.Builder(getActivity());
+        builder.setTitle(R.string.post_settings_status);
+        int checkedItem = getCurrentPostStatusIndex();
+        // Current index should never be -1, but if if is, we don't want to crash
+        if (checkedItem == -1) {
+            checkedItem = 0;
+        }
+        builder.setSingleChoiceItems(R.array.post_settings_statuses, checkedItem, null);
+        builder.setPositiveButton(R.string.dialog_button_ok, new DialogInterface.OnClickListener() {
```

Comment:
We [recently started](https://github.com/wordpress-mobile/WordPress-Android/pull/5651) using `android.R.string.ok`, can you switch it here?